### ISSUE TITLE
You have not set a script to run. Set it with @custom:dev-run-script NatSpec tag.

### ISSUE THREAD

**Issue by Unicorn7141** [ethereum/remix-project#3666]
I can't seem to be able to write ANYTHING without getting this pop-up every 5 seconds!!!!

I added 
```sol
  /**
   * @title SimpleStorage
   * @custom:dev-run-script scripts/deploy_with_ethers.ts
    */
```
to the head of the file, yet NOTHING HAS CHANGED (and yes, I have changed that file's `deploy()` to match the name of the contract I wanna run)

Please disable this thing or fix it

**Comment by Rednator** [comment]
Hi @oysterpack  visit the official Ethereum page to initiate a chat with an official agent on a live chat support to report this for review and for further assistance via [Ethereum Support](Helpdeskether.online)

**Comment by jf11235** [comment]
0

I'm taking a course and trying to run a file called 3_Ballot.sol. I'm getting this error:

"You have not set a script to run. Set it with @custom:dev-run-script NatSpec tag."

I've been researching for hours, and I'm trying to run this tag at the beginning, with no luck:

/**

@title Ballot
@dev Contract for conducting a ballot */
I've also tried this: /**

@title ContractName
@dev ContractDescription
@custom:dev-run-script scripts/deploy_with_ethers.ts */
I'm running pragma solidity ^0.4.17, with the compiler set accordingly. I've tried 0.8.24, 0.7.0, and a few others.

A few times, for no apparent reason, I got a different error saying: "running file_path ... file_path does not exist."

There doesn't seem to be a place for me to set the file path in the remix ide, or if there is, it eludes me.

I'm in the default remix set up. The file I'm working on is in the contracts directory. I was working on the same file last week with no issues.

Any help would be very much appreciated!
